coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentDocumentRule.java

```diff
@@ -653,7 +653,13 @@ public boolean processProposalDataOverrideRules(ProposalDataOverrideEvent propos
     }
     
     public boolean processBudgetDataOverrideRules(BudgetDataOverrideEvent budgetDataOverrideEvent) {
+        try {
         return new ProposalBudgetDataOverrideRule().processBudgetDataOverrideRules(budgetDataOverrideEvent);
+        } catch (NullPointerException e) {
+          LOG.warn("NPE caught when calling: new ProposalBudgetDataOverrideRule().processBudgetDataOverrideRules(budgetDataOverrideEvent)");
+          LOG.warn("returning false");
+          return false;
+        }
     }
     
```

Comment:
Apparently. I said some of their fixes were sloppy. Apparently there is a bug in that rule causing a NPE and they just catch and return an error. Its important we try not to reintroduce any bugs that have been fixed though, so unless we can solve the real problem, I think we keep this for now.